chapters/synchronization.txt

```diff
@@ -1231,7 +1231,7 @@ One example of this would be if the pending signal operation is, itself,
 waiting on the same semaphore at a lower value and the call to
 fname:vkSignalSemaphore signals that lower value.
 Furthermore, if there are two or more processes or threads signaling the
-same timeline semaphore from the host, the client must ensure that the
+same timeline semaphore from the host, the application must: ensure that the
```

Comment:
Policy is not to use normative terms in Notes, so this should just be "the application must ensure". There are cases where we've done it, but they are bugs and we should remove them.